Neos.Eel/Classes/FlowQuery/OperationInterface.php

```diff
@@ -42,7 +42,7 @@ public static function isFinal();
      * can work with the $context objects. It can be implemented
      * to implement runtime conditions.
      *
-     * @param array (or array-like object) $context onto which this operation should be applied
+     * @param $context array (or array-like object) $context onto which this operation should be applied
```

Comment:
Should'nt that be `@param array $context`?

Reply:
or `@param array|iterable $context`

Reply:
yes good catch ;)